configuration/raspi-config.md

```diff
@@ -2,7 +2,7 @@
 
 This page describes the console based raspi-config application. If you are using the Raspberry Pi desktop then you can use the graphical Raspberry Pi Configuration application from the Preferences menu to configure your Raspberry Pi.
 
-`raspi-config` is the Raspberry Pi configuration tool originally written by [Alex Bradbury](https://github.com/asb). It targets Raspberry Pi OS.
+`raspi-config` is the Raspberry Pi configuration tool originally written by [Alex Bradbury](https://github.com/asb) and lives at [github.com/RPi-Distro/raspi-config](https://github.com/RPi-Distro/raspi-config/tree/master/raspi-config). It targets Raspberry Pi OS.
```

Comment:
Perhaps "which now lives at" ?
Also, not sure we need the full tree and branch info, rather than just a link to the repo?

Reply:
I don;t like the phrase "lives at" either. "Is hosted at" or similar?